assets/benchmarks/shaders/CMakeLists.txt

```diff
@@ -99,3 +99,38 @@ generate_rules_for_shader("shader_benchmark_texture"
     SOURCE "${PPX_DIR}/assets/benchmarks/shaders/Benchmark_Texture.hlsl"
     INCLUDES "${PPX_DIR}/assets/benchmarks/shaders/VsOutput.hlsli"
     STAGES "ps")
+
+generate_rules_for_shader("shader_foveation_benchmark"
+    SOURCE "${PPX_DIR}/assets/benchmarks/shaders/FoveationBenchmark.hlsl"
+    INCLUDES "${PPX_DIR}/assets/benchmarks/shaders/VsOutput.hlsli"
+    STAGES "vs" "ps")
+
+generate_rules_for_shader("shader_foveation_benchmark_shading_rate_khr"
+    SOURCE "${PPX_DIR}/assets/benchmarks/shaders/FoveationBenchmarkShadingRateKHR.hlsl"
+    INCLUDES "${PPX_DIR}/assets/benchmarks/shaders/VsOutput.hlsli"
+    STAGES "ps")
+
+add_custom_command(
+    OUTPUT "${PPX_DIR}/assets/benchmarks/shaders/spv/FoveationBenchmarkFragSizeEXT.vs.spv"
+    MAIN_DEPENDENCY "${PPX_DIR}/assets/benchmarks/shaders/FoveationBenchmarkFragSizeEXT.vert"
+    COMMAND ${CMAKE_COMMAND} -E echo "Compiling ps FoveationBenchmarkFragSizeEXT.vert to spv/FoveationBenchmarkFragSizeEXT.vs.spv"
+    COMMAND "${GLSLC_PATH}" -o "${PPX_DIR}/assets/benchmarks/shaders/spv/FoveationBenchmarkFragSizeEXT.vs.spv" "${PPX_DIR}/assets/benchmarks/shaders/FoveationBenchmarkFragSizeEXT.vert")
```

Comment:
should we add a function for this, similar to generate_rules_for_shader? Or maybe change it so it can take glsl?